<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExistedCfgDialog</class>
 <widget class="QDialog" name="ExistedCfgDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>已存在应用配置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="shadowFrame">
     <property name="styleSheet">
      <string notr="true">#shadowFrame{
background-color:white;
border:none;
border-radius:12px;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QGroupBox" name="groupBoxLeft">
         <property name="styleSheet">
          <string notr="true">#groupBoxLeft{
font: 700 14pt &quot;幼圆&quot;;
}</string>
         </property>
         <property name="title">
          <string>已配置应用列表</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QLabel" name="labelJsonErr">
            <property name="cursor">
             <cursorShape>IBeamCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 0, 0);</string>
            </property>
            <property name="text">
             <string>Error!</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="listView">
            <property name="frameShape">
             <enum>QFrame::Box</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnAddProcessCfg">
              <property name="text">
               <string>新建</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnRefreshCfgFilePath">
              <property name="text">
               <string>刷新</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnOpenCfgFilePath">
              <property name="text">
               <string>打开配置文件路径</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_5">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="frameRight">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QGroupBox" name="groupBoxRight">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>200</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">#groupBoxRight{
font: 700 14pt &quot;幼圆&quot;;
}</string>
            </property>
            <property name="title">
             <string>配置信息</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="text">
                  <string>别名(习惯称呼)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelAlias">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>22</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>IBeamCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color:#F0F0F0;
padding-left:2px;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>进程名称</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelProcessName">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>22</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>IBeamCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color:#F0F0F0;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>完整路径</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelProcessPath">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>22</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>IBeamCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color:#F0F0F0;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>启动等待时间</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QSpinBox" name="spinBoxWatingTime">
                 <property name="minimumSize">
                  <size>
                   <width>60</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximum">
                  <number>999</number>
                 </property>
                 <property name="value">
                  <number>5</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="text">
                  <string>秒(s)</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>参数列表</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QTextEdit" name="textEditProcessParam">
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="text">
                  <string>窗口类名</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelClassName">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>22</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>IBeamCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color:#F0F0F0;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>窗口标题</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelWndTitle">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>22</height>
                  </size>
                 </property>
                 <property name="cursor">
                  <cursorShape>IBeamCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">background-color:#F0F0F0;</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="wordWrap">
                  <bool>true</bool>
                 </property>
                 <property name="textInteractionFlags">
                  <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <property name="rightMargin">
                <number>8</number>
               </property>
               <item>
                <spacer name="horizontalSpacer_4">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QCheckBox" name="checkBoxRestart">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>如果进程已经启动，杀掉进程后重启，否则直接抓取窗口</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QToolButton" name="btnModify">
              <property name="text">
               <string>修改</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnDelete">
              <property name="text">
               <string>删除</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>138</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="btnAdd2StartPage">
              <property name="text">
               <string>添加到启动配置</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxStartEmbeded">
              <property name="text">
               <string>并且执行启动嵌入</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listView</tabstop>
  <tabstop>btnAdd2StartPage</tabstop>
  <tabstop>checkBoxStartEmbeded</tabstop>
  <tabstop>btnAddProcessCfg</tabstop>
  <tabstop>btnRefreshCfgFilePath</tabstop>
  <tabstop>btnOpenCfgFilePath</tabstop>
  <tabstop>spinBoxWatingTime</tabstop>
  <tabstop>textEditProcessParam</tabstop>
  <tabstop>checkBoxRestart</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
